game/scripts/npc/abilities/death_prophet_spirit_siphon.txt

```diff
@@ -70,7 +70,7 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "charge_restore_time"                             "60 52 44 36 28 16"
+        "charge_restore_time"                             "36 32 28 24 20 16" //OAA
```

Comment:
Here is an idea, make 36 sec charge restore time at every level. Its not op in lategame and its buffed in early and mid game. What do you think?